src/Resources/views/GalleryAdmin/list.html.twig

```diff
@@ -18,7 +18,7 @@ file that was distributed with this source code.
                 {{ "label.select_context"|trans({}, 'SonataMediaBundle') }}
                 <div class="btn-group">
                     <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                        <strong class="text-info">{{ persistent_parameters.context }}</strong> <span class="caret"></span>
+                        <strong class="text-info">{{ persistent_parameters.context|trans({}, 'SonataMediaBundle') }}</strong> <span class="caret"></span>
```

Comment:
How is this related to the fix?

Reply:
Sorry, I did not see that. It was part of a pull request from December that I thought were already merged... https://github.com/sonata-project/SonataMediaBundle/pull/1345 ?

Reply:
Do you know how to get rid of this with `git rebase -i`?